src/components/tooltip/TooltipDirective.js

```diff
@@ -490,12 +491,21 @@ goog.require('ga_topic_service');
               var name = feature.get('name');
               var featureId = feature.getId();
               var layerId = feature.get('layerId') || layer.get('bodId');
+              if (layer.get('type') == 'KML') {
+                layerId = layer.label;
+                if (name && name.length) {
+                  featureId = name;
+                }
+              }
               var id = layerId + '#' + featureId;
               htmlpopup = htmlpopup.
                   replace('{{id}}', id).
                   replace('{{descr}}', feature.get('description') || '').
                   replace('{{name}}', (name) ? '(' + name + ')' : '');
               feature.set('htmlpopup', htmlpopup);
+              if (!isFeatureQueryable(feature)) {
```

Comment:
If you want the same behavior in 3d you remove the test here : https://github.com/geoadmin/mf-geoadmin3/blob/369cb88b9c9aa3e341d2f6af1bbbce2318c6be50/src/components/tooltip/TooltipDirective.js#L347